utils/atomic-storybook/.storybook/register.js

```diff
@@ -1,9 +1,9 @@
-import React from 'react';
-import {addons, types} from '@storybook/manager-api';
 import {AddonPanel} from '@storybook/components';
-import {ShadowPartPanel} from './shadow-parts-addon/shadow-parts-panel';
-import {CodeSamplePanel} from './code-sample-addon/code-sample-panel';
+import {addons, types} from '@storybook/manager-api';
 import {debounce} from 'lodash';
+import React from 'react';
+import {CodeSamplePanel} from './code-sample-addon/code-sample-panel';
+import {ShadowPartPanel} from './shadow-parts-addon/shadow-parts-panel';
```

Comment:
Prettier like stuff organized.

Reply:
not sure why these files weren't getting formatted until now 🤔 glad it's formatted now 🤷